<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect><x>0</x><y>0</y><width>800</width><height>600</height></rect>
  </property>
  <property name="windowTitle"><string>Bus App</string></property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item><widget class="QLineEdit" name="lineEdit"/></item>
    <item><widget class="QComboBox" name="comboBox"/></item>
    <item><widget class="QLabel" name="imageLabel"><property name="text"><string>Image</string></property></widget></item>
    <item><widget class="QTableWidget" name="tableWidget"/></item>
    <item><widget class="QListWidget" name="listWidget"/></item>
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item><widget class="QPushButton" name="btnAdd"><property name="text"><string>Добавить</string></property></widget></item>
      <item><widget class="QPushButton" name="btnRemove"><property name="text"><string>Удалить</string></property></widget></item>
      <item><widget class="QPushButton" name="btnSortAsc"><property name="text"><string>Сорт ↑</string></property></widget></item>
      <item><widget class="QPushButton" name="btnSortDesc"><property name="text"><string>Сорт ↓</string></property></widget></item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>